controllers/usersController.js

```diff
@@ -3,16 +3,17 @@ const { getAllUsers, getUserById } = require('../queries/usersQueries');
 
 const usersController = express.Router();
 
-usersController.get('/', (req, res) => {
-  users = getAllUsers() || [];
-  res.status(200).json({ data: users });
+// Routes
+usersController.get('/', async (req, res) => {
+  const users = await getAllUsers();
+  res.status(200).json(users);
```

Comment:
The response here is not consistent with the healthcheck, which returns `{ data: "some string"}`
And in this controller there is no `data` key.
I don't have a strong opinion about how you do it, but I think it is good for all responses to return the same format.